lahja/asyncio/endpoint.py

```diff
@@ -216,9 +222,13 @@ def __init__(
         endpoint has acknowledged the new subscription set. If ``block`` is ``False`` then this
         function will return immediately after the send finishes.
         """
```

Comment:
The comment just above this line isn't quite right anymore: "Alert the ``Endpoint`` which has connected to us that our subscription set has changed." Since `RemoteEndpoint` now handles both inbound and outbound this should be something like "Alert the remote that this endpoint's subscription set has changed"